github/repos.go

```diff
@@ -65,6 +65,7 @@ type Repository struct {
 	HasIssues         *bool   `json:"has_issues"`
 	HasWiki           *bool   `json:"has_wiki"`
 	HasPages          *bool   `json:"has_pages"`
+	HasProjects       *bool   `json:"has_projects"`
```

Comment:
@ungureanuvladvictor can you please add `,omitempty` as per this comment - https://github.com/google/go-github/pull/782#discussion_r151298759 ?